HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/ViewLocationPopup/ViewLocationPopup.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@using HelpMyStreet.Utils.Extensions;
+@using HelpMyStreet.Utils.Enums;
+
+@model HelpMyStreetFE.Models.Account.Jobs.ViewLocationViewModel
+
+
+<div class="popup__content">
+    <partial name="_PopupCloseButton" />
+    @if (Model.IsAllowed)
+    {
+        <div id="location-popup" class="popup__content__wrapper popup-map">
+            <div class="popup__content__header">Where is the help needed?</div>
+            <div class="text-left">
+                <p class="mb-sm">Help is needed in <strong>@Model.PostCode</strong>.</p>
+                <p class="mt-sm">This is @Model.Distance miles away from your registered address; for directions, click on the map icon below.</p>
+            </div>
+            <div id="map-@Model.encodedJobID" style="width:100%; height:150px" class="location-map" data-lat="@Model.Coordinates.Latitude" data-lng="@Model.Coordinates.Longitude">
+            </div>
+            <div class="popup__content__buttons mt-md">
+                <p class="error dnone"></p>
+                <partial name="_PopupAcceptButton" model='new PopupButtonModel { ButtonText = "Close" }' />
+            </div>
+        </div>
+    }
+    else
```

Comment:
We'll never hit the else case currently.  Either remove the IsAllowed parameter, or populate it with false in the ViewComponenet instead of throwing an Exception.